basic/src/test/java/org/jboss/hal/testsuite/test/configuration/logging/CustomFormatterTestCase.java

```diff
@@ -65,12 +69,27 @@ public static void tearDown() throws IOException, OperationException, TimeoutExc
     @Page
     private LoggingPage page;
 
+    private BackupAndRestoreAttributes backupAndRestoreAttributes;
+
     @Before
-    public void before() {
+    public void before() throws CommandFailedException {
         page.navigate();
         page.switchToFormatterTab();
         page.switchToCustomPattern();
         page.selectFormatter(CUSTOM_FORMATTER);
+        backupAndRestoreAttributes = new BackupAndRestoreAttributes.Builder(CUSTOM_FORMATTER_ADDRESS).build();
+        client.apply(backupAndRestoreAttributes.backup());
+    }
+
+
+
+    @After
+    public void restoreFormatterAndGetRidOfReloadRequiredWindow() throws CommandFailedException, InterruptedException, TimeoutException, IOException {
+        if (backupAndRestoreAttributes != null) {
+            client.apply(backupAndRestoreAttributes.restore());
+        }
+        Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent();
+        administration.reloadIfRequired();
```

Comment:
`administration.reloadIfRequired()` should be sufficient and `Console.withBrowser(browser).dismissReloadRequiredWindowIfPresent()` is not necessary since console is refreshed before each `CustomFormatterTestCase` test.